cryptol-remote-api/src/CryptolServer/Modules.hs

```diff
@@ -52,14 +52,15 @@ visibleModules _ = concatMap (moduleToInfos False) . loadedModules <$> getModule
 
 moduleToInfos :: PP a => Bool -> T.ModuleG a -> [VisibleModuleInfo]
 moduleToInfos p m =
+  let p' = p || isParametrizedModule m in
```

Comment:
Possibly dumb question: now that we explicitly check `isParameterizedModule m` on each recursive call, do we need to pass the `Bool` argument anymore? The only place where the `Bool` argument changes is on each recursive call to each of the `mFunctors` (where `True` is passed instead of `False`), but I would expect `isParametrizedModule` to always return `True` on nested functors anyway.